core/src/labels/textLabel.cpp

```diff
@@ -123,4 +123,8 @@ void TextLabels::setQuads(std::vector<GlyphQuad>& _quads) {
     style.context()->lockAtlas(m_atlasRefs);
 }
 
+size_t TextLabels::bufferSize() const {
+    return style.meshBufferSize();
+}
+
```

Comment:
TextLabels has no GL memory usage. This counts the current frame LabelMeshes for each tile?